geomstats/information_geometry/information_manifolds.py

```diff
@@ -0,0 +1,170 @@
+"""Mixin for manifolds of probability distributions."""
+
+from scipy.integrate import quad
+
+import geomstats.backend as gs
+import geomstats.errors
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class InformationManifold:
+    """Mixin for manifolds of probability distributions."""
+
+    def sample(self, point, n_samples=1):
+        """Sample from the probability distribution.
+
+        Sample from the probability distribution with parameters provided
+        by point. This gives n_samples points.
+
+        Parameters
+        ----------
+        point : array-like, shape=[..., dim]
+            Point representing a probability distribution.
+        n_samples : int
+            Number of points to sample for each set of parameters in point.
+            Optional, default: 1.
+
+        Returns
+        -------
+        samples : array-like, shape=[..., n_samples]
+            Sample from the probability distributions.
+        """
+        raise NotImplementedError("The sample method is not yet implemented.")
+
+    def _hard_pdf(self, point, x):
```

Comment:
I don't believe we need this. Because the information manifolds that inherit from InformationManifold will "hard code" the method `pdf`, in _their_ `pdf` method, if they need to.